src/pages/croptype.vue

```diff
@@ -0,0 +1,38 @@
+<template>
+  <q-page padding>
+
+     <q-btn
+      padding="xl"
+      color="secondary"
+      round
+      icon="eco" no-caps :to="{ path: 'vegelist' }"
+    />
+
+     <q-btn
+      padding="xl"
+      color="secondary"
+      round
+      icon="eco" no-caps :to="{ path: 'fruitlist' }"
+
+    />
+ 
+    <q-btn
+      padding="xl"
+      color="secondary"
+      round
+      icon="eco" no-caps :to="{ path: 'flowerlist' }"
+
+    />
+
+    <q-btn :ripple="{ center: true }" 
+    color="secondary" 
+    label="Back" no-caps :to="{ path: 'harvesttype' }" /> 
```

Comment:
I think the Malay version of "Back" in this context, should be "Semula" or "Balik". Or swap the `label` for `icon` with a left arrow. Whichever you decide, apply the same across all Back buttons.